vividus-tests/src/main/resources/story/integration/JSON.story

```diff
@@ -227,3 +227,30 @@ Then JSON `
   "required": [ "productId", "productName", "price", "tags" ]
 }
 `
+
+Scenario: Validate anyOf json-unit matcher
+Then JSON element from `${json}` by JSON path `$.store` is equal to `
+{
+  "book":"#{json-unit.matches:anyOf}#{escapeJson(
```

Comment:
please add 2 tests: with escapeJSON and without it

Reply:
done:
1) inlined 
2) escaped